resources/views/partials/route.blade.php

```diff
@@ -12,7 +12,11 @@
 > Example request:
 
 @foreach($settings['languages'] as $language)
+@if(view()->exists("apidoc::partials.example-requests.$language"))
```

Comment:
Please remove this line. It's better for the user to receive an error that view "java" does not exist, rather than failing silently.